presto-native-execution/presto_cpp/main/http/HttpClient.h

```diff
@@ -123,7 +124,7 @@ class HttpClient {
   folly::EventBase* const eventBase_;
   const folly::SocketAddress address_;
   const folly::HHWheelTimer::UniquePtr timer_;
-
+  const std::string_view type_;
```

Comment:
Why not define this a enum class?

Reply:
I think we can even not have enum. The constant is not giong to be reused anywhere else. We can just have string literal passing in ctor.

Reply:
Use enum class is a common practice here. We can have kindToString() to convert it to a string for display.